src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -558,12 +558,21 @@ def trusted_launch_warning_log(namespace, generation_version, features):
                   'Launch VM by default. To know more about the default change and Trusted Launch, ' \
                   'please visit https://aka.ms/TLaD'
 
+    upgrade_hint = 'Consider upgrading security for your workloads using Azure Trusted Launch VMs. ' \
+                   'To know more about Trusted Launch, please visit ' \
+                   'https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch.'
```

Comment:
Could we consider placing these identical warning logs into a global static constant? This can avoid duplicate definitions

Reply:
> Could we consider placing these identical warning logs into a global static constant _(such as a new file `_constants.py`)_? This can avoid duplicate definitions

That makes sense👍